airflow/providers/google/cloud/example_dags/example_dataflow.py

```diff
@@ -52,6 +52,7 @@
 GCS_OUTPUT = os.environ.get("GCP_DATAFLOW_GCS_OUTPUT", "gs://INVALID BUCKET NAME/output")
```

Comment:
Those files (in google/cloud/example_dags/) are meant to be migrated to the system tests and then removed. Can you verify if dataflow tests are already migrated and if so modify the migrated file and remove those ones?

Reply:
System tests was migrated but at the moment I couldn't do required changes even after rebase of the branch